Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs

```diff
@@ -49,6 +50,26 @@ internal abstract class AnalyticsDataHandlerBase {
         protected virtual int version { get; }
         protected virtual int eventsPerHour => 10000;
 
+        protected abstract Dictionary<string, DateTime> lastSendTimes { get; }
+
+        protected virtual TimeSpan MinTimeBetweenEvents {
+            get { return TimeSpan.FromSeconds(10); }
```

Comment:
This should be 1 second, not 10 seconds right?

Reply:
It's 10 seconds in Colm's PR. Can set it to anything we want though. How much do we need to reduce the analytics by?

Reply:
I think we are supposed to send an event every 1 second